chromium_src/.clangd

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Disable unused includes check in chromium_src.
```

Comment:
Curious why this is needed.

Reply:
> Curious why this is needed.

sometimes we just need to add some includes so our `#define` won't break other things. These includes are frequently marked as "unused" which is noisy. We cannot fix this, so just disable this check for all files inside `brave/chromium_src`.

I land this as a part of this PR just because I hit it again and decided to finally fix it.